strawberry/types/nodes.py

```diff
@@ -95,6 +97,7 @@ def from_node(cls, info: GraphQLResolveInfo, node: GQLFragmentSpreadNode):
         fragment = info.fragments[name]
         return cls(
             name=name,
+            python_name=camel_to_snake(node.name.value),
```

Comment:
I think this might not be the right way to go, since it doesn't reflect the configuration options.

We can have auto camel case off, which might not be an issue for this case, but we can also have a custom name converter.

Do you think we can find a way to get the strawberry type here and access the python name from there?

Reply:
Yeah, I was about to write something about... 
In this way is more a guess: everything is lowercase when for example the Child types can be Title Case... etc. 

So I'm digging in the Node object if there is something that can help us, but I didn't find much... 
I was thinking find the field from the schema:

```
schema: Schema = info.schema._strawberry_schema  # type: ignore
        field: StrawberryField = schema.get_field_for_type(  # type: ignore
            field_name=node.name.value,
            type_name=info.parent_type.name,
        )
```
But I'm not sure I have the right parent_type... 🤔

Reply:
yeah, I just missing the `parent_type`, if I had that I could get the field from the schema... but I don't know where to find it unfortunately :(

Reply:
`info.parent_type.extensions['strawberry-definition']` this should be what you need :)

Reply:
The thing is info is always the same, and is not changing between the nodes so even in `info.parent_type.extensions['strawberry-definition']` there is always the `Query` root type and one field... I would like to find the other fields... every node should have its field somewhere...  🤔 
right?

Reply:
ah, that's annoying, I'll take a look at it tomorrow and see how we can do this